src/js/components/ResultItem.jsx

```diff
@@ -51,9 +51,6 @@ export class ResultItem extends Component {
             const statusGlyph = getGlyphFor(resultClean);
             this.setState({resultClean, statusGlyph});
         }
-        if (props.expanded!=this.props.expanded) {
```

Comment:
it was checking the incoming `props.expanded != this.props.expanded`. Why not just change it to `this.state.expanded != props.expanded`? And thank you for being:
![image](https://cloud.githubusercontent.com/assets/3009477/16820266/378fed56-491e-11e6-88b3-85b5db8448b6.png)